docs/usage.asciidoc

```diff
@@ -164,6 +164,14 @@ _Default:_ `null`
 |`querystring`
 |`object` - Custom querystring for the request. +
 _Default:_ `null`
+
+|`id`
+|`any` - Custom request id. _(overrides the top level request id generator)_ +
+_Default:_ `null`
+
+|`context`
+|`any` - Custom object per request. _(you can use it to pass some data to the clients events)_ +
```

Comment:
Is this passed as is and never modified/cloned? I'm trying to make sure that there will be some way for people to establish a cache in a `WeakMap` for each request.

Reply:
It is never cloned. I try to avoid cloning as much as possible since it will lead to bad performances.